### PR TITLE
add margin to accordion

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR add the option to change how the margin sizes are being applied to the accordion panel heading
#### Where should the reviewer start?
accordionPanel.js
#### What testing has been done on this PR?
storybook
#### How should this be manually tested?
storybook
#### Any background context you want to provide?
When a label was passed into accordionPanel there was no say for the users on what size the margin around it was. This was all done within the AccordionPanel.js starting at Line 70
There was really no way for the users to customize what they wanted for heading. 

The margin was being taken from the `Heading line size` for level 4
#### What are the relevant issues?
design system
#### Do the grommet docs need to be updated?
auto
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

**Comment by britt6612** [comment]
@ShimiSun moved to a new branch sorry last branch was a merge mess!

**Comment by ShimiSun** [comment]
>still not sure where the snapshot changes are coming from in the theme I did see your PR though addressing this

No worries, we'll be taken a look at those on a separate PR. I'll ignore those files while reviewing.

File: src/js/components/AccordionPanel/doc.js [hunk 1]
```diff
@@ -41,6 +46,11 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: '<FormDown />',
   },
+  'accordion.pad': {
+    description: 'The pad size around the accordion.',
```
Comment (ShimiSun) [inline src/js/components/AccordionPanel/doc.js:50]: The description here isn't accurate, I ran some tests and it looks like the placement of the pad is only wrapping the heading and not the pad of the Accordion itself.
From this description, I would expect the accordion to have a pad, but the pad only applies on the wrapper of the heading and not even on a Panel.

Was that the intention of this PR? I would think the design-system would want the pad to apply on the all Panel.

See an example I created:
image

Comment (britt6612) [reply src/js/components/AccordionPanel/doc.js:50]: yes this is around the heading. It is set at 'horizontal: xsmall' which we are not able to change in theme and design system the pad around header is a little big bigger.

Comment (ShimiSun) [reply src/js/components/AccordionPanel/doc.js]: Cleaned the pad out, resolving the conversation.

File: src/js/components/Accordion/stories/Custom.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const CustomAccordionTheme = {
   accordion: {
     heading: {
       level: 3,
+      margin: { left: '6px', right: '6px', top: '24px', bottom: '24px' },
```
Comment (ShimiSun) [inline src/js/components/Accordion/stories/Custom.js:12]:
```suggestion
      margin: { vertical: '6px', horizontal: '24px' },
```

**Pull request merged**